ckanext/geocat/utils/xpath_utils.py

```diff
@@ -211,10 +222,39 @@ def xpath_get_url_from_node(node):
 
 def xpath_get_distribution_from_distribution_node(
         resource_node, protocol):
```

Comment:
Here I felt it needed a comment to describe what protocols exist and how they influence the mapping.